File: python/1074.py
# -*- coding: utf-8 -*-

n = int(input())
for _ in range(1, n+1):
    v = int(input())
    if v%2==0 and v>0:
        print('EVEN POSITIVE')
    elif v%2==0 and v<0:
        print('EVEN NEGATIVE')
    elif v%2!=0 and v>0:
        print('ODD POSITIVE')
    elif v%2!=0 and v<0:
        print('ODD NEGATIVE')
    elif v==0:
        print('NULL')

File: categorias/iniciante/c++/1074.cpp
#include <iostream>

using namespace std;

int main() {
    int Ns[10000], i, n;

    cin >> n;
    for (i=0;i<n;i++) {
        cin >> Ns[i];
    }
    for (i=0;i<n;i++) {
        if (Ns[i] == 0) {
            cout << "NULL\n";
        } else if (Ns[i] % 2 == 0) {
            cout << "EVEN";
        } else {
            cout << "ODD";
        }
        if (Ns[i] > 0) {
            cout << " POSITIVE\n";
        } else if (Ns[i] < 0) {
            cout << " NEGATIVE\n";
        }
    }

    return 0;
}